service/src/test/java/org/databiosphere/workspacedataservice/controller/ConcurrentDataTypeChangesTest.java

```diff
@@ -41,33 +47,46 @@ class ConcurrentDataTypeChangesTest extends TestBase {
   private static final RecordType recordType = RecordType.valueOf("concurrency");
   private static final String versionId = "v0.2";
 
+  @Autowired private TwdsProperties twdsProperties;
+
   @BeforeEach
-  void setUp() {
+  void setUp() throws JsonProcessingException {
+    CollectionId collectionId = CollectionId.of(UUID.randomUUID());
+    String name = "test-name";
+    String description = "test-description";
+
+    CollectionServerModel collectionServerModel = new CollectionServerModel(name, description);
+    collectionServerModel.id(collectionId.id());
+
+    ObjectMapper objectMapper = new ObjectMapper();
+
     headers = new HttpHeaders();
     headers.setContentType(MediaType.APPLICATION_JSON);
-    instanceId = UUID.randomUUID();
     ResponseEntity<String> response =
         restTemplate.exchange(
-            "/instances/{v}/{instanceid}",
+            "/collections/v1/{workspaceId}",
             HttpMethod.POST,
-            new HttpEntity<>("", headers),
+            new HttpEntity<>(objectMapper.writeValueAsString(collectionServerModel), headers),
             String.class,
-            versionId,
-            instanceId);
+            twdsProperties.workspaceId().id());
     assertEquals(HttpStatus.CREATED, response.getStatusCode());
+
+    JsonNode jsonNode = objectMapper.readTree(response.getBody());
+
+    instanceId = UUID.fromString(jsonNode.get("id").asText());
```

Comment:
I suggest using strongly-typed objects here instead of `JsonNode`
```suggestion
    CollectionServerModel created =
        objectMapper.readValue(response.getBody(), CollectionServerModel.class);

    instanceId = created.getId();
```